### ISSUE TITLE
Doesn't have May/June 2020 Past Papers

### ISSUE THREAD

**Issue by Yamgirly** [micromaomao/schsrch#8]
I like paper.sc, but its a shame that i have to fall back to papacambridge all the time cos you guys don't have M/J 2020 past papers. It would be nice if you added those :)

**Comment by micromaomao** [comment]
Hi, sorry about that, but the downloader I use to get past papers from Cambridge's school support website is currently broken, and due to university and other commitments I haven't had time to fix it. I will try to have a go at it again sometime soon.

Also, if you happen to have access to a teacher account there and some nodejs debugging skills, feel free to contribute a fix to the downloader in this project, that would also help me get those paper faster and I would highly appreciate it.

I'm glad you liked paper.sc, thanks for your understanding!

**Comment by Yamgirly** [comment]
Nah, I sadly don't have a teacher account. I noticed you have a papacambridge downloader, what about that one. I know that papacambridge typically have quite recent exams (including the M/J 2020 exams)

**Comment by etanx** [comment]
I might be able to gain access to a teacher account if that helps, but unfortunately cannot help with nodejs debugging. ;'(
